src/qz/auth/Certificate.java

```diff
@@ -79,6 +81,23 @@
 
     static {
         try {
+            Properties sslProperties = DeployUtilities.loadSSLProperties();
+            String certFile = sslProperties.getProperty("ca.certificate");
```

Comment:
If the installer is writing this property, it should already be the full path, thus avoiding all of the working directory logic, then the logic in `overridePath` can be reused without the redundant block.